packages/admin/src/NotificationManager.php

```diff
@@ -35,6 +35,9 @@ public function handleLivewireResponses(Component $component, Response $response
         session()->forget('notifications');
 
         if (count($notifications) > 0) {
+            if (! Livewire::isLivewireRequest()) {
```

Comment:
Why is the check twice? The method wont reach this part if the first check fails

Reply:
Yeah I was asking the same 😅 Probably Cmd + V at the wrong time and didn't notice.